stubs/Database/ManagesTransactions.stub

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Illuminate\Database\Concerns;
+
+trait ManagesTransactions
+{
+    /**
+     * Execute a Closure within a transaction.
+     *
+     * @template TCallbackReturnType
+     *
+     * @param  \Closure($this): TCallbackReturnType  $callback
+     * @param  int  $attempts
```

Comment:
You could make this positive-int as making 0 attempts makes no sense

Reply:
Good idea 👍